developer-guides/doc_guidelines.rst

```diff
@@ -79,6 +80,9 @@ directive to point to other documents that may include additional ``toctree``
 directives of their own, ultimately collecting all the content into an
```

Comment:
directives? or directories?

Reply:
toctree directives (it's a ReStructuredText term).